ivy_tests/test_ivy/test_functional/test_nn/test_activations.py

```diff
@@ -116,16 +116,17 @@ def test_sigmoid(*, dtype_and_x, test_flags, backend_fw, fn_name, on_device):
 @handle_test(
     fn_tree="functional.ivy.softmax",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
         min_num_dims=1,
         large_abs_safety_factor=8,
-        small_abs_safety_factor=8,
+        small_abs_safety_factor=4,
         safety_factor_scale="log",
     ),
     axis=st.one_of(
         helpers.ints(min_value=-1, max_value=0),
         st.none(),
     ),
+    test_gradients=st.just(False),
```

Comment:
Was this to resolve the error paddle gives about gradients? If so, I think you need to remove it because that's a wider issue with ivy as a whole rather than this particular function

Reply:
I changed it already I think I that line was added cuz softmax in torch doesn't support bfloat16 dtype which caused some erros but I  had removed it already.